server/service/src/invoice_line/stock_out_line/insert/mod.rs

```diff
@@ -73,7 +78,17 @@ pub fn insert_stock_out_line(
         .connection
         .transaction_sync(|connection| {
             let (item, invoice, batch) = validate(&input, &ctx.store_id, connection)?;
-            let (new_line, update_batch) = generate(connection, input, item, batch, invoice)?;
+
+            // Check if we need to override the pricing with a default
+            let pricing = get_pricing_for_item(
```

Comment:
maybe can go in generate method (for in downstream PR)

Reply:
Added too https://github.com/msupply-foundation/open-msupply/issues/4763